proto/ibc/lightclients/wasm/v1/query.proto

```diff
@@ -8,36 +8,36 @@ option go_package = "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/type
 
 // Query service for wasm module
 service Query {
-  // Get all Wasm code IDs
-  rpc CodeIds(QueryCodeIdsRequest) returns (QueryCodeIdsResponse) {
-    option (google.api.http).get = "/ibc/lightclients/wasm/v1/code_ids";
+  // Get all Wasm code hashes
+  rpc CodeHashes(QueryCodeHashesRequest) returns (QueryCodeHashesResponse) {
+    option (google.api.http).get = "/ibc/lightclients/wasm/v1/code_hashes";
   }
 
-  // Get Wasm code for given code ID
+  // Get Wasm code for given code hash
   rpc Code(QueryCodeRequest) returns (QueryCodeResponse) {
-    option (google.api.http).get = "/ibc/lightclients/wasm/v1/code_ids/{code_id}/code";
+    option (google.api.http).get = "/ibc/lightclients/wasm/v1/code_hashes/{code_hash}/code";
   }
 }
 
-// QueryCodeIdsRequest is the request type for the Query/CodeIds RPC method.
-message QueryCodeIdsRequest {
+// QueryCodeHashesRequest is the request type for the Query/CodeHashes RPC method.
+message QueryCodeHashesRequest {
   // pagination defines an optional pagination for the request.
   cosmos.base.query.v1beta1.PageRequest pagination = 1;
 }
 
-// QueryCodeIdsResponse is the response type for the Query/CodeIds RPC method.
-message QueryCodeIdsResponse {
-  repeated string code_ids = 1;
+// QueryCodeHashesResponse is the response type for the Query/CodeHashes RPC method.
+message QueryCodeHashesResponse {
+  repeated string code_hashes = 1;
   // pagination defines an optional pagination for the request.
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
 // QueryCodeRequest is the request type for the Query/Code RPC method.
 message QueryCodeRequest {
-  string code_id = 1;
+  string code_hash = 1;
```

Comment:
Code ID has a different meaning in `x/wasm` so I thought of renaming to `code_hash` to avoid confusion between `x/wasm` and `08-wasm`.

Reply:
how is the meaning different? i thought they both referred to stored contract code?

Reply:
Sorry, "meaning" is probably not the right word... What I wanted to say is that the term code ID in `x/wasm` is an integer that indeed is used to refer to the contract ([this integer is generated when a contract is stored](https://github.com/CosmWasm/wasmd/blob/788cdbbfc3788b97799cf9c075cd4c4e5e4bbb8a/x/wasm/keeper/keeper.go#L175-L183)) and the code hash is the hash of the byte code. So I thought that to avoid confusion between the code ID in `x/wasm` and the code ID we were using here it was better to rename it to code hash, so that we use the same terms for the same things in both modules.